dcmgr/Gemfile

```diff
@@ -30,7 +30,7 @@ gem 'isono', '>= 0.2.20'
 ENV['PATH']="#{Bundler.bundle_path}/bin:#{ENV['PATH']}"
 
 gem 'rant', '0.5.9'
-gem 'trema', :github=>'axsh/trema', :branch=>'wakame'
+# gem 'trema', :github=>'axsh/trema', :branch=>'wakame'
```

Comment:
Is it safe to remove trema at this time? It's definitely something we want to remove but I'm not sure if NATbox still relies on it or not.